src/txn/txn_rollback_to_stable.c

```diff
@@ -718,11 +724,16 @@ __rollback_abort_ondisk_kv(WT_SESSION_IMPL *session, WT_REF *ref, WT_ROW *rip, u
       !__rollback_txn_visible_id(session, vpack->tw.start_txn) ||
       (!WT_TIME_WINDOW_HAS_STOP(&vpack->tw) && prepared)) {
         __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-          "on-disk update aborted with start durable timestamp: %s, commit timestamp: %s, "
-          "prepared: %s, stable timestamp: %s and txnid : %" PRIu64,
-          __wt_timestamp_to_string(vpack->tw.durable_start_ts, ts_string[0]),
-          __wt_timestamp_to_string(vpack->tw.start_ts, ts_string[1]), prepared ? "true" : "false",
-          __wt_timestamp_to_string(rollback_timestamp, ts_string[2]), vpack->tw.start_txn);
+          "on-disk update with commit timestamp %s aborted because start durable timestamp (%s) > "
```

Comment:
We have a function that prints the entire time point `__wt_time_point_to_string`. How do you feel printing the time point at one place and the comparison results later?

Reply:
@kommiharibabu, do you mean something like this:
```
         __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-          "on-disk update with commit timestamp %s aborted because start durable timestamp (%s) > "
-          "stable timestamp "
-          "(%s): %s, or txnid (%" PRIu64
-          ") is not visible: %s, or tw has not stop and is prepared: %s",
-          __wt_timestamp_to_string(vpack->tw.start_ts, ts_string[0]),
-          __wt_timestamp_to_string(vpack->tw.durable_start_ts, ts_string[1]),
-          __wt_timestamp_to_string(rollback_timestamp, ts_string[2]),
-          vpack->tw.durable_start_ts > rollback_timestamp ? "true" : "false", vpack->tw.start_txn,
+          "on-disk update aborted with time window %s. Start durable timestamp > stable timestamp: "
+          "%s, or txnid is not visible: %s, or tw has not stop and is prepared: %s",
+          __wt_time_point_to_string(
+            vpack->tw.start_ts, vpack->tw.durable_start_ts, vpack->tw.start_txn, time_string),
+          vpack->tw.durable_start_ts > rollback_timestamp ? "true" : "false",
           !__rollback_txn_visible_id(session, vpack->tw.start_txn) ? "true" : "false",
           !WT_TIME_WINDOW_HAS_STOP(&vpack->tw) && prepared ? "true" : "false");
```

Reply:
Yes. This way the time window of an on-disk entry or update is printed at the start of the message and the result decision is later. I feel It avoids mistakes.

Reply:
Thanks, pushed as part of 7ce9cc1d825443b90fe45ed3070265a8458ecf8c